Source/Redemption/Public/RedemptionPlayer.h

```diff
@@ -110,12 +118,5 @@ class REDEMPTION_API ARedemptionPlayer : public ACharacter
 	void StartCrouch();
 	void EndCrouch();
 
-public:	
-	virtual void Tick(float DeltaTime) override;
-
-	virtual void SetupPlayerInputComponent(class UInputComponent* PlayerInputComponent) override;
 
-public:
-	FORCEINLINE class USpringArmComponent* GetCameraBoom() const { return CameraBoom; }
-	FORCEINLINE class UCameraComponent* GetFollowCamera() const { return TPPCamera; }
```

Comment:
you changed it to be based on our code standards thank you

Reply:
^-^b